src/footer.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import logo from './media/staticLogo.svg';
+
+function Footer() {
+    
+    const scrollTop = () => {
+        window.scrollTo({top: 0, behavior: 'smooth'});
+    };
+
+    return (
+        <footer className="flex  flex-row justify-between bg-footer text-white absolute left-0 right-0 bottom-0 font-light font-opensans" id="footer-container">
+           
+            <div className='flex flex-row justify-between w-full p-4 ' id="content-container">
+                <div className='flex flex-col justify-between text-left'id="left-content">
+                    <div className="px-4 flex justify-left" id="logo-container">
+                        <img  src={logo} className="w-24 h-20 " alt="logo" />
```

Comment:
Can we link the logo to the home page? i.e. if I click the logo while I'm on the about page, it takes me to the home page